shared/utils.js

```diff
@@ -42,7 +42,7 @@ var LOG_LEVELS = {
     ALL: Number.MAX_SAFE_INTEGER
 };
 
-var LOG_LEVEL = LOG_LEVELS.ALL;
+var LOG_LEVEL = LOG_LEVELS.INFO;
```

Comment:
That value was being changed to info or debug in createApp - so the config parsing was done with full debugging.